enterprise_subsidy/apps/content_metadata/api.py

```diff
@@ -246,9 +246,8 @@ def get_content_metadata(content_identifier, **kwargs):
                 course_details,
                 django_cache_timeout=CONTENT_METADATA_CACHE_TIMEOUT,
             )
-        logger.info(
-            f'Fetched course details {course_details} for content_identifier {content_identifier}',
-        )
```

Comment:
We log this several hundred times a day, but it's the _entire_ content metadata blob each time.
Instead, let's log only when we couldn't fetch a given identifier.